packages/lisk-p2p/src/peer_discovery.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { Peer } from './peer';
+
+export interface DiscoverPeersReturntype {
```

Comment:
For return types, we should try to have more meaningful names. If another method elsewhere in the code also needs to return an object in the form `{inbound: [], outbound: []}` (now or in the future), it would be better if they had the same type; no need to couple to a specific method.

Maybe we can call this `PeerAdjacencyInfo`, `PeerConnectionSchema`, `PeerRelationInfo`, `PeerRelationSchema` or something like this.

Reply:
I like `PeerRelationSchema`, sounds appropriate